### ISSUE TITLE
As someone making a transaction, I would like to see the transaction fee in the expanded view

### ISSUE THREAD

**Issue by adjwood** [strideynet/ModelWorld-Bank#163]
So right now it looks like this, with the TX Fee showing up as a separate transaction

![image](https://user-images.githubusercontent.com/12470650/72349584-af493a80-36d4-11ea-8b32-034b34c9ea3a.png)

I think it would be nice to have it look something like this:

![image](https://user-images.githubusercontent.com/12470650/72349555-a35d7880-36d4-11ea-8e99-1d6ec1dab614.png)

A bit less obstructive to the user to see who they have actually sent money to but at the same time it still allows them to see the transaction fee

**Comment by Padanub** [comment]
Agree, go a long way to decluttering TXs?

**Comment by strideynet** [comment]
This is technically quite complex in terms of how to surface this info to the users

**Comment by Padanub** [comment]
Technically complex in the sense of a "not worth the time" complex?

**Comment by strideynet** [comment]
No, more just in, there needs to be some clever thinking here rather than it's a 5 minute job.

**Comment by Padanub** [comment]
Couldn't we do it the way we do income? So the tx fee is a sub of the outgoing transaction?

**Comment by strideynet** [comment]
@Padanub Depends if you want NubBank to receive the TX fees.

**Comment by Padanub** [comment]
If the answer is we can do this if Nub Bank doesn't receive the tx fees i'm kinda fine with that tbf, for UX reasons. NUb Bank really doesn't need the cash

**Comment by strideynet** [comment]
Hmm. It's still a bit of a bitch. I think the tx will still technically exist, we can just hide them.
